lib/github_fetcher.rb

```diff
@@ -51,6 +51,10 @@ def language
     team_config['language']
   end
 
+  def repo
+    "#{ORGANISATION}/#{team_config['repo']}"
```

Comment:
The original project ours was forked from was built by a British person :) We can certainly change it, but I would do that in a separate PR since the naming isn't new to this PR.